sources/packages/web/src/services/AppConfigService.ts

```diff
@@ -75,10 +82,75 @@ export class AppConfigService {
   }
 
   async initAuthService(type: ClientIdType) {
+    if (this.authService) {
+      return;
+    }
     if (this._config) {
+      this._authClientType = type;
       this.authService = await AuthService(this._config, type);
     } else {
       throw new Error("Unable to load application: server is not responding");
     }
   }
+
+  roles(): string[] {
```

Comment:
If we are not using this code, please consider removing it.

Reply:
DONE